tests/benchmark_test_inputs/Constraint_net_zero_energy/csv_elements/constraints.csv

```diff
@@ -0,0 +1,5 @@
+,unit,constraints
+minimal_renewable_factor,factor,0
+maximum_emissions,kgCO2eq/a,None
+minimal_degree_of_autonomy,factor,0
+net_zero_energy,bool,True
```

Comment:
I think I would go for more folders than less; just out of the reason that having a input data set for each simulation is more explicit than changing parameters in the code (this is why in the end I decided for 4 folders instead of 2 although it was a very tight decision :D) 
I'll rename the folders.

Reply:
done with  6ea58d6